java/src/server/facade/ServerFacade.java

```diff
@@ -808,6 +817,41 @@ public CommandExecutionResult getModel(int gameID, int version) throws GetModelE
         return new CommandExecutionResult(jsonString);
     }
 
+    /**
+     * retrieves games and users from the database and populates the server.  This function should be run when
+     * ServerFacade is constructed so that it can start out with up to date information
+     */
+    private void importData() {
+        IDatabase database = Config.database;
```

Comment:
Does the DatabaseFacade inherit from IDatabase now? If so, awesome. If not, we gotta change some things